### ISSUE TITLE
alicloud_logtail_config create is broken and keeps updating in-place

### ISSUE THREAD

**Issue by fabien-duolingo** [aliyun/terraform-provider-alicloud#4194]
### Terraform Version
```
Terraform v0.12.31
+ provider.alicloud v1.138.0
```

### Affected Resource(s)
- [alicloud_logtail_config](https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/logtail_config)

### Terraform Configuration Files
```hcl
resource "alicloud_log_project" "log_project" {
  name        = "test-459ebaa7e431c5b8795a410c" 
  description = "Managed by Terraform"
}

resource "alicloud_log_store" "log_store" {
  project               = alicloud_log_project.log_project.name
  name                  = "test-logstore"
  auto_split            = true
  max_split_shard_count = 64
  retention_period      = 180
}

resource "alicloud_log_machine_group" "log_machine_group" {
  project       = alicloud_log_project.log_project.name
  name          = "test-haproxies"
  identify_type = "ip"
  identify_list = [
      "10.16.18.117",
      "10.16.18.118"
  ]
}

resource "alicloud_logtail_config" "logtail_config" {
  project      = alicloud_log_project.log_project.name
  logstore     = alicloud_log_store.log_store.name
  input_type   = "file"
  name         = "haproxy-json"
  output_type  = "LogService"
  input_detail = <<DEFINITION
  {
      "adjustTimezone": false,
      "fileEncoding": "utf8",
      "filePattern": "haproxy.log",
      "localStorage": true,
      "logPath": "/var/log",
      "logType": "json_log",
      "plugin": {
        "global": {
          "DefaultLogQueueSize": 10
        },
        "processors": [
          {
            "detail": {
              "SplitSep": "",
              "SplitKey": "content"
            },
            "type": "processor_split_log_string"
          },
          {
            "detail": {
              "SourceKey": "content",
              "ExpandDepth": 1,
              "ExpandConnector": "_"
            },
            "type": "processor_json"
          },
          {
            "detail": {
              "Anchors": [
                {
                  "ExpondJson": true,
                  "Start": "]:",
                  "Stop": "",
                  "FieldName": "haproxy",
                  "FieldType": "json"
                }
              ],
              "SourceKey": "content"
            },
            "type": "processor_anchor"
          }
        ]
      }
  }
  DEFINITION
}

resource "alicloud_logtail_attachment" "logtail_attachment" {
  project             = alicloud_log_project.log_project.name
  logtail_config_name = alicloud_logtail_config.logtail_config.name
  machine_group_name  = alicloud_log_machine_group.log_machine_group.name
}
```

### Expected Behavior
1. The Logtail configuration is created according to the terraform code.
2. After apply, the next `terraform plan` shows "No changes. Infrastructure is up-to-date."

### Actual Behavior
1. Creation succeeds but on the console several differences can be noticed on the Configuration Details, Advanced Options : 
- **Enable Plug-in Processing** is disabled (should be enabled). 
- **Plug-in Config** is the default and not the one specified by `input_detail`. 
2. Terraform wants to update in-place the `alicloud_logtail_config` resource :
```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # alicloud_logtail_config.logtail_config will be updated in-place
  ~ resource "alicloud_logtail_config" "logtail_config" {
        id           = "test-459ebaa7e431c5b8795a410c:test-logstore:haproxy-json"
      ~ input_detail = jsonencode(
          ~ {
                adjustTimezone = false
                fileEncoding   = "utf8"
                filePattern    = "haproxy.log"
                localStorage   = true
                logPath        = "/var/log"
                logType        = "json_log"
              + plugin         = {
                  + global     = {
                      + DefaultLogQueueSize = 10
                    }
                  + processors = [
                      + {
                          + detail = {
                              + SplitKey = "content"
                              + SplitSep = ""
                            }
                          + type   = "processor_split_log_string"
                        },
                      + {
                          + detail = {
                              + ExpandConnector = "_"
                              + ExpandDepth     = 1
                              + SourceKey       = "content"
                            }
                          + type   = "processor_json"
                        },
                      + {
                          + detail = {
                              + Anchors   = [
                                  + {
                                      + ExpondJson = true
                                      + FieldName  = "haproxy"
                                      + FieldType  = "json"
                                      + Start      = "]:"
                                      + Stop       = ""
                                    },
                                ]
                              + SourceKey = "content"
                            }
                          + type   = "processor_anchor"
                        },
                    ]
                }
            }
        )
        input_type   = "file"
        logstore     = "test-logstore"
        name         = "haproxy-json"
        output_type  = "LogService"
        project      = "test-459ebaa7e431c5b8795a410c"
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

### Steps to Reproduce
1. Use the code above to create a new Logtail config (`terraform apply`). 
2. After creation, run `terraform plan` or `terraform apply`.
Even if we apply the second plan, any further `terraform apply` will produce the same output. 

### References
*Possibly* relates to #3824 and #3866.

**Comment by HuiJing-C** [comment]
Do you mean that some of the attributes after `terraform apply` do not meet expectations, and "No changes. Infrastructure is up-to-date." is displayed when using `terraform plan` or `terraform apply` after modifying the configuration again?

**Comment by fabien-duolingo** [comment]
@HuiJing-C two things are happening : 
1. Yes some of the attributes do not meet expectations, like `input_detail`. 
2. `No changes. Infrastructure is up-to-date.` is what is expected after `terraform plan` or `terraform apply`, but instead it wants to apply changes (update in-place) forever (they are never really applied).

**Comment by HuiJing-C** [comment]
@fabien-duolingo Ok, wait for us to test

**Comment by fabien-duolingo** [comment]
Thanks, you should be able to reproduce by using the terraform code I provided.

**Comment by HuiJing-C** [comment]
@fabien-duolingo ok, the problem has been located, we will try to fix it this week

**Comment by HuiJing-C** [comment]
@fabien-duolingo i'm very sorry, this problem has been fixed in version 1.145.0

**Comment by fabien-duolingo** [comment]
@HuiJing-C I can confirm that it now works as expected, thank you for fixing this!
